src/Form.php

```diff
@@ -323,6 +343,25 @@ public function setAttributes(array $values): void
         }
     }
 
+    public function validate(): bool
+    {
+        $this->clearErrors();
+
+        if (!empty($this->attributesRules)) {
+            $results = (new Validator($this->attributesRules))->validate($this);
+
+            foreach ($results as $attribute => $result) {
+                if ($result->isValid() === false) {
+                    foreach ($result->getErrors() as $error) {
```

Comment:
`$this->addErrors([$attribute => $result->getErrors()]);`

Reply:
Done.